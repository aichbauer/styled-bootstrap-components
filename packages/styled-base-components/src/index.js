import { BootstrapBaseCss } from './components';

export { BootstrapBaseCss };
