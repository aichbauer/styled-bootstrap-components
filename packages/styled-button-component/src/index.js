import {
  Button,
  LinkButton,
} from './components/Button';
import { ButtonGroup } from './components/ButtonGroup';
import { ButtonToolbar } from './components/ButtonToolbar';

export {
  Button,
  LinkButton,
  ButtonGroup,
  ButtonToolbar,
};
