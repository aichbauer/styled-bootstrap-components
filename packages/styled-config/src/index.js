import { theme } from './theme';
import {
  border,
  borderRadius,
  boxShadow,
  colors,
  fontFamily,
  fontSize,
  fontWeight,
  height,
  margin,
  padding,
  screenSize,
  width,
} from './utils';

export {
  theme,
  // utils
  border,
  borderRadius,
  boxShadow,
  colors,
  fontFamily,
  fontSize,
  fontWeight,
  height,
  margin,
  padding,
  screenSize,
  width,
};
