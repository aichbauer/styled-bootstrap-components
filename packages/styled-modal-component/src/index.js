import { Modal } from './components/Modal';
import { ModalWrapper } from './components/ModalWrapper';
import { ModalBody } from './components/ModalBody';
import { ModalContent } from './components/ModalContent';
import { ModalDialog } from './components/ModalDialog';
import { ModalFooter } from './components/ModalFooter';
import { ModalHeader } from './components/ModalHeader';
import { ModalTitle } from './components/ModalTitle';
import { ModalBackdrop } from './components/ModalBackdrop';

export {
  Modal,
  ModalWrapper,
  ModalBody,
  ModalContent,
  ModalDialog,
  ModalFooter,
  ModalHeader,
  ModalTitle,
  ModalBackdrop,
};
