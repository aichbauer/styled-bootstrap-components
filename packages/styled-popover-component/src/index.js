import { PopoverWithoutTransition, Popover } from './components/Popover';
import { PopoverArrow } from './components/PopoverArrow';
import { PopoverBody } from './components/PopoverBody';
import { PopoverHeader } from './components/PopoverHeader';

export {
  PopoverWithoutTransition,
  Popover,
  PopoverArrow,
  PopoverBody,
  PopoverHeader,
};
