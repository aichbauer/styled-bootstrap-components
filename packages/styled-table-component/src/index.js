import { Table } from './components/Table';
import { Tr } from './components/Tr';

export {
  Table,
  Tr,
};
