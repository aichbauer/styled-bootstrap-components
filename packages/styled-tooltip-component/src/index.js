import { TooltipWithoutTransition, Tooltip } from './components/Tooltip';
import { TooltipArrow } from './components/TooltipArrow';
import { TooltipInner } from './components/TooltipInner';

export {
  TooltipWithoutTransition,
  Tooltip,
  TooltipArrow,
  TooltipInner,
};
