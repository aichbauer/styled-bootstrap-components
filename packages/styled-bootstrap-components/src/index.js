import { Alert } from 'styled-alert-component';
import { Badge } from 'styled-badge-component';
import {
  A,
  Abbr,
  Address,
  Article,
  Aside,
  B,
  Blockquote,
  Button as BaseButton,
  Caption,
  Code,
  Dd,
  Dfn,
  Dialog,
  Dl,
  Dt,
  Fieldset,
  Figcaption,
  Figure,
  Footer,
  H1,
  H2,
  H3,
  H4,
  H5,
  H6,
  Header,
  Hgroup,
  Hr,
  Img,
  Input,
  Kbd,
  Legend,
  Label,
  Main,
  Nav as Navigation,
  Ol,
  Optgroup,
  Output,
  P,
  Pre,
  Progress,
  Samp,
  Section,
  Select,
  Small,
  Strong,
  Sub,
  Summary,
  Sup,
  Svg,
  Table as BaseTable,
  Textarea,
  Th,
  Ul,
} from 'styled-base-components';
import {
  Breadcrumb,
  BreadcrumbItem,
} from 'styled-breadcrumb-component';
import {
  Button,
  ButtonGroup,
  ButtonToolbar,
} from 'styled-button-component';
import {
  Card,
  CardBody,
  CardFooter,
  CardHeader,
  CardImageHeader,
  CardText,
  CardTitle,
} from 'styled-card-component';
import { screenSize } from 'styled-config';
import { Container } from 'styled-container-component';
import {
  Dropdown,
  DropdownItem,
  DropdownMenu,
  DropdownDivider,
} from 'styled-dropdown-component';
import {
  FormCheck,
  FormCheckInput,
  FormControl,
  FormControlFile,
  FormControlPlainText,
  FormGroup,
  FormInline,
  FormText,
  InputGroup,
  InputGroupAppend,
  InputGroupPrepend,
  InputGroupText,
} from 'styled-form-component';
import {
  Column,
  Row,
} from 'styled-grid-system-component';
import { Jumbotron } from 'styled-jumbotron-component';
import {
  ListGroup,
  ListGroupItem,
} from 'styled-listgroup-component';
import {
  Modal,
  ModalBody,
  ModalContent,
  ModalDialog,
  ModalFooter,
  ModalHeader,
  ModalTitle,
} from 'styled-modal-component';
import {
  Nav,
  NavLink,
} from 'styled-nav-component';
import {
  Navbar,
  NavbarLink,
} from 'styled-navbar-component';
import {
  Popover,
  PopoverArrow,
  PopoverBody,
  PopoverHeader,
} from 'styled-popover-component';
import {
  Table,
  Tr,
} from 'styled-table-component';
import {
  Tooltip,
  TooltipArrow,
  TooltipInner,
} from 'styled-tooltip-component';

export {
  Alert,
  Badge,
  A,
  Abbr,
  Address,
  Article,
  Aside,
  B,
  Blockquote,
  BaseButton,
  Caption,
  Code,
  Dd,
  Dfn,
  Dialog,
  Dl,
  Dt,
  Fieldset,
  Figcaption,
  Figure,
  Footer,
  H1,
  H2,
  H3,
  H4,
  H5,
  H6,
  Header,
  Hgroup,
  Hr,
  Img,
  Input,
  Kbd,
  Legend,
  Label,
  Main,
  Navigation,
  Ol,
  Optgroup,
  Output,
  P,
  Pre,
  Progress,
  Samp,
  Section,
  Select,
  Small,
  Strong,
  Sub,
  Summary,
  Sup,
  Svg,
  BaseTable,
  Textarea,
  Th,
  Ul,
  Breadcrumb,
  BreadcrumbItem,
  Button,
  ButtonGroup,
  ButtonToolbar,
  Card,
  CardBody,
  CardFooter,
  CardHeader,
  CardImageHeader,
  CardText,
  CardTitle,
  screenSize,
  Container,
  Dropdown,
  DropdownItem,
  DropdownMenu,
  DropdownDivider,
  FormCheck,
  FormCheckInput,
  FormControl,
  FormControlFile,
  FormControlPlainText,
  FormGroup,
  FormInline,
  FormText,
  InputGroup,
  InputGroupAppend,
  InputGroupPrepend,
  InputGroupText,
  Column,
  Row,
  Jumbotron,
  ListGroup,
  ListGroupItem,
  Modal,
  ModalBody,
  ModalContent,
  ModalDialog,
  ModalFooter,
  ModalHeader,
  ModalTitle,
  Nav,
  NavLink,
  Navbar,
  NavbarLink,
  Popover,
  PopoverArrow,
  PopoverBody,
  PopoverHeader,
  Table,
  Tr,
  Tooltip,
  TooltipArrow,
  TooltipInner,
};
