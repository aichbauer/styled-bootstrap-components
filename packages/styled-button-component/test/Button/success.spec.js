import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Button } from '../../src';

test('Styles matches Button success disabled', () => {
  const tree = renderer.create(<Button success disabled />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Styles matches Button success', () => {
  const tree = renderer.create(<Button success />).toJSON();
  expect(tree).toMatchSnapshot();
});
