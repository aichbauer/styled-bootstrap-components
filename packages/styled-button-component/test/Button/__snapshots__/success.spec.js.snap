// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles matches Button success 1`] = `
.c0 {
  font-weight: 400;
  line-height: 1.5;
  text-align: center;
  white-space: nowrap;
  vertical-align: middle;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-transition: color 0.15s ease-in-out,background-color 0.15s ease-in-out,border-color 0.15s ease-in-out,box-shadow 0.15s ease-in-out;
  transition: color 0.15s ease-in-out,background-color 0.15s ease-in-out,border-color 0.15s ease-in-out,box-shadow 0.15s ease-in-out;
  border: 1px solid #28a745;
  background-color: #28a745;
  border-radius: 0.25rem;
  display: inline-block;
  font-size: 1rem;
  color: #fff;
  padding: 0.375rem 0.75rem;
}

.c0:focus {
  outline: 0;
  box-shadow: 0 0 0 0.2rem rgba(40,167,69,0.5);
}

.c0:hover,
.c0:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
  border: 1px solid #28a745;
  background-color: #28a745;
  color: #fff;
  cursor: pointer;
}

.c0:hover:focus,
.c0:focus:focus,
.c0:hover:hover,
.c0:focus:hover {
  border: 1px solid #1e7e34;
}

.c0:hover:focus,
.c0:focus:focus,
.c0:hover:hover,
.c0:focus:hover {
  background-color: #218838;
}

.c0:focus,
.c0:hover {
  border: 1px solid #1e7e34;
}

.c0:focus,
.c0:hover {
  background-color: #218838;
}

<button
  className="c0"
/>
`;

exports[`Styles matches Button success disabled 1`] = `
.c0 {
  font-weight: 400;
  line-height: 1.5;
  text-align: center;
  white-space: nowrap;
  vertical-align: middle;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-transition: color 0.15s ease-in-out,background-color 0.15s ease-in-out,border-color 0.15s ease-in-out,box-shadow 0.15s ease-in-out;
  transition: color 0.15s ease-in-out,background-color 0.15s ease-in-out,border-color 0.15s ease-in-out,box-shadow 0.15s ease-in-out;
  border: 1px solid #28a745;
  background-image: none;
  background-color: #28a745;
  border-radius: 0.25rem;
  display: inline-block;
  opacity: 0.65;
  font-size: 1rem;
  color: #fff;
  padding: 0.375rem 0.75rem;
}

.c0:focus {
  outline: 0;
  box-shadow: 0 0 0 0.2rem rgba(40,167,69,0.5);
}

.c0:hover,
.c0:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
  border: 1px solid #28a745;
  background-image: none;
  background-color: #28a745;
  color: #fff;
}

<button
  className="c0"
  disabled={true}
/>
`;
